### PR TITLE
features/pageerror_500

### PR DESCRIPTION
**Pull request by Alextran04121998**
[Index] Error page  500

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -83,6 +83,16 @@ app.use(function(err, req, res, next) {
...
   res.render('error');
+
+// 500 error page
+  if (404 === err.status && req.app.get('env') === 'production') {
+    res.writeHead(302, {'Location': '/404'});
+    res.end();
+  }
+
+  res.status(err.status || 500);
+  res.render('error');
```
Comment (trantanhdev) [inline app.js:94]: remove these code

File: app.js [hunk 2]
```diff
@@ -83,6 +83,16 @@ app.use(function(err, req, res, next) {
...
+  res.sendFile(path.join(__dirname, 'public/pages/500.html'));
```
Comment (trantanhdev) [inline app.js:95]: put this to replace line 89 and 90

File: app.js (hunk 1 above)
Comment (Alextran04121998) [reply app.js:94]: oki a

File: public/pages/500.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+<body>
+<style type="text/css">
+    #main {
+        height: 100vh;
+    }
+</style>
+<div class="d-flex justify-content-center align-items-center" id="main">
+    <h1 class="mr-3 pr-3 align-top border-right inline-block align-content-center">500</h1>
+    <div class="inline-block align-middle">
+        <h2 class="font-weight-normal lead" id="desc">The page you requested was not found.</h2>
```
Comment (trantanhdev) [inline public/pages/500.html:19]: Change message "The page you requested was not found." to "The page not working"

File: public/pages/404.html [hunk 4]
```diff
@@ -21,4 +21,5 @@ <h2 class="font-weight-normal lead" id="desc">The page you requested was not fou
     </div>
 </div>
 </body>
-</html>
\ No newline at end of file
+</html>
+
```
Comment (trantanhdev) [inline public/pages/404.html:25]: remove this change

**Pull request closed without merging**